examples/channel.rb

```diff
@@ -6,6 +6,14 @@
 
 on_connect_error = -> { puts 'on connect error got called' }
 
+on_connection_timeout = -> {puts 'connection timeout happened'}
+
+on_connection_dropped = -> {puts 'connection dropped from existing sockets'}
+
+on_connection_encrypted = -> {puts 'Connection is encrypted from server, check the encryption key received'}
+
+on_connection_status_changed = -> {put 'Connection status has changed'}
```

Comment:
puts here

Reply:
@sacOO7 Can you please run a rubocop -a over this file as I see a lot of style-linting errors.